remotebackup.sh

```diff
@@ -1,12 +1,26 @@
 #! /bin/bash
 
-#This script connects  to a remote server via ssh, creates a .tar.gz backup, and uses scp to secure copy it to the host computer. 
-# $1 is username, $2 is the server address, $3 is the remote directory to be backed up, and $4 is the name of the backupfile
 
+#Example  use:
+#./remotebackup.sh username remoteip /directory/to/backup nameofbackupfile
+
+# Arguments 
+# $1 username of remote user
+# $2 ip address of remote server
+# $3 remote directory to be backed up (e.g. /var/www/bob)
+# $4 is the name of the .tar.gz backup (e.g. bobbackup) 
+# End of arguments 
```

Comment:
I don't think it's conventional.  The guy who teaches my online course  about shell scripting does it so  I just  did it too.  If you think it's superfluous, I'll get rid of it.